test/nested.jl

```diff
@@ -4,14 +4,20 @@
   @test nugget(γ) ≈ 0.3
   @test sill(γ) ≈ 1.0
   @test range(γ) ≈ 50.0
+  @test (@elapsed sill(γ)) < 100e-6
```

Comment:
Shoudln't this be `1e-6`? I remember the new times were in ns (1e-9) right? We could make the test even tighter with `1e-7` for example.